js/windowMgr.js

```diff
@@ -410,18 +417,20 @@ function doCreateMainWindow(initialUrl, initialBounds, isCustomTitleBar) {
 
             let webContents = newWinOptions.webContents;
 
+            // Event needed to hide native menu bar
+            webContents.once('did-start-loading', () => {
+                let browserWin = BrowserWindow.fromWebContents(webContents);
+                if (isWindowsOS && browserWin && !browserWin.isDestroyed()) {
```

Comment:
can you explain the logic behind this code? And how does it help solve this issue?

Reply:
This logic just hides the menu bar as soon as the window starts loading instead of setting it to null.

Setting the menu to `null` on Windows the accelerators will be cleared and the keyboard shortcut doesn't work.